tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.yaml

```diff
@@ -1,5 +1,5 @@
 context:
-  paginator_page:
+  pages:
```

Comment:
I think the yaml isn't quite correct here as it isn't showing the current page (and we need to add it to the blog listing page too) but I can fix that in the front-end ticket I've created for this.